sysutils/rtx/Portfile

```diff
@@ -0,0 +1,336 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           cargo 1.0
+
+github.setup        jdxcode rtx 1.35.8 v
+revision            0
```

Comment:
please add the "github.tarball_from" filed and set it to "releases" (preferred if upstream provides those), otherwise "archive"

Reply:
rtx does not use releases for distributing source code; therefore, select "archive" instead.